packages/web3/package.json

```diff
@@ -20,7 +20,9 @@
     "types": "types/index.d.ts",
     "scripts": {
         "tsc": "tsc -b tsconfig.json",
-        "dtslint": "dtslint --localTs ../../node_modules/typescript/lib types"
+        "dtslint": "dtslint --localTs ../../node_modules/typescript/lib types",
+        "preinstall": "echo \"WARNING: the web3-shh and web3-bzz api will be deprecated in the next version\"",
```

Comment:
Do we need both pre and post? I think just post could suffice, no?

Reply:
it was mentioned in slack that that was wanted can remove though if it's overkill